test/_external/test262/skiplist

```diff
@@ -99,3 +105,33 @@ test/language/module-code/instn-star-props-nrml.js
 
 # NEEDS INVESTIGATION!
 test/language/module-code/parse-err-reference.js
+
```

Comment:
Many of these fails look to be related to having `cjs.mutableNamespace` enabled 👇

Reply:
ah. ok. that's good tho! what's the plan forward to make them pass? is that with `mjs` disabled as well? I would assume `mjs` and `use module` work the same (or similar)?

Reply:
to avoid `.mjs` you could just include ` .esmrc` or `.esmrc.json` in the test files folder or parent folder with specific options disabled.